rpcutil/policygen/policygen.go

```diff
@@ -29,28 +49,42 @@ func main() {
 		&cluster.PeerMonitorRPCAPI{},
 	}
 
-	fmt.Println(`
+	fmt.Fprintln(os.Stderr, `
 // The below generated policy keeps the endpoint types
-// from the exiting one, marking new endpoints as NEW. Copy-paste this policy
-// into rpc_auth.go and set the NEW endpoints to their correct type.`)
-	fmt.Println()
+// from the existing one, marking new endpoints as NEW. Redirect stdout
+// into rpc_policy.go and set the NEW endpoints to their correct type (make
+// sure you have recompiled this binary with the current version of the code).
+============================================================================`)
+	fmt.Fprintln(os.Stderr)
 
-	fmt.Println("var RPCPolicy = map[string]RPCEndpointType{")
+	fmt.Println("package ipfscluster")
```

Comment:
I suggest writing this all to a byte slice and then running [`format.Source`](https://golang.org/pkg/go/format/#Source) for format the output. This would prevent compile time errors from wiping out the existing `rpc_policy.go` file as happened when I was testing this and the unused `fmt` import caused `go fmt` to bork (I ran `make install`).

Reply:
Thanks, that's a way better way